src/Query.jl

```diff
@@ -1,9 +1,9 @@
 module QueryLib
 
-export Ports, Query, make_query,
+export Query, make_query,
   Ob, Hom, dom, codom, compose, ⋅, ∘, id, otimes, ⊗, munit, braid, σ,
   dagger, dunit, dcounit, mcopy, Δ, delete, ◊, mmerge, ∇, create, □,
-  meet, top, FreeBicategoryRelations, @program,
+  meet, top, FreeBicategoryRelations, @program, @relation,
```

Comment:
looks like we don't need to be importing `@program` any more because we just use UWDs.

Reply:
I was thinking about keeping the `@program` functionality in there in case users want to use a more imperative programming style. But keeping it does end up in a lot of redundant unnecessary code, which may not be worth maintaining.

Reply:
Oh I see, there is an imperative syntax if you use the hypergraph category morphisms as opposed to the relation syntax. That is interesting, especially if we think of traces which would allow iteration in the queries, which would support queries like connected components.